### PR TITLE
SAK-42643 - Switching from fonticonpicker to fontawesome-iconpicker f…

### PR DESCRIPTION
**Pull request by jonespm**
…or keyboard accessibility and updates

I didn't see any useful updates to the previous picker and it was not keyboard accessible. 

This is currently what the new one looks like
image

The old plugin when called turned a input text field into a field that looked like this 
image

That seems like it's going to take a little more work with this plugin if we want to achieve that but adding our own custom callbacks. Not sure how important it is to be accessible verses working and looking like it did before. 

I tried to get it to look like the "inside dropdowns" view of https://jonespm.github.io/fontawesome-iconpicker/ but seemed like it would require changing all of the current input's over to a hidden, adding a button AND modifying the callback to update the hidden value. Whenever I just had it a button with the value updated there wasn't anything posted. . . 
 
Happy for feedback or someone else to commit on top of this. I might go back to experimenting with the button again.

### CODE REVIEW

**Comment by jonespm** [comment]
I updated this to look like the below in the code. I think this is fine, just will need to check the other tools that use this and need to figure out an issue with tabbing. 

image

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- library/src/webapp/js/headscripts.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5185681&bid=17014493)

**Comment by jonespm** [comment]
Okay this *looks* good. There's one more issue where if you are navigating with the keyboard and get to the icon picker, after you select an icon the focus goes to a random place on the page rather than to the next element.

I'm having to call this to get the focus into the input
`event.iconpickerInstance.popover.find('input').focus()`

Maybe it will have to focus back on the actual input again? Or the one after? Will have to look at that but maybe some other ideas?

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- library/src/webapp/js/headscripts.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5185681&bid=17014493)

File: library/pom.xml [hunk 1]
```diff
@@ -294,6 +294,13 @@
 			<artifactId>viewerjs</artifactId>
 			<version>0.5.8</version>
 		</dependency>
+		<dependency>
```
Comment (mpellicer) [inline library/pom.xml]: Would you mind documenting in what tools this webjar is used? There are some examples in this file.

Thanks again! I've approved it.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- library/src/webapp/js/headscripts.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5185681&bid=17014493)

**Pull request merged**